### PR TITLE
Complementary palette in KD view mode

### PR DESCRIPTION
**Pull request by scavnickyj**
- replace some ::before icons with Icon component
- update shadow color
- update KPI styles
- style mobile vertical menu, lists and dropdowns

JIRA: BB-2819

---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2881/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4512/ : FAILURE in 12m 17s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2882/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4513/ : SUCCESS in 12m 31s

**Comment by scavnickyj** [comment]
extended test

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/739/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2883/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/739/ : UNSTABLE in 22m 42s

**Comment by scavnickyj** [comment]
^^ unrelated

File: libs/sdk-ui-ext/styles/internal/scss/scheduled_mail_recipients.scss [hunk 1]
```diff
@@ -21,6 +21,10 @@
         transition-duration: 0.25s;
         transition-timing-function: ease-in-out;
         border-radius: 3px;
+
+        &:hover {
+            border-color: var(--gd-palette-complementary-5, rgb(179, 179, 179));
```
Comment (yanlipnican) [inline libs/sdk-ui-ext/styles/internal/scss/scheduled_mail_recipients.scss:26]: How about we change `rgb(179, 179, 179)` to `#b3b3b3` to be consistent

**Review by yanlipnican — APPROVED** [review]

**Comment by yanlipnican** [comment]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2901/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4530/ : SUCCESS in 12m 48s

**Review by yanlipnican — APPROVED** [review]

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/849/ : FAILURE in 3m 11s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1113/ : SUCCESS in 26s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4531/ : SUCCESS in 12m 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/223/ : FAILURE in 29s (non-voting)

**Pull request merged**